packages/integration-sdk-http-client/src/index.ts

```diff
@@ -19,6 +20,7 @@ export type APIResourceIterationCallback<T> = (
 ) => boolean | void | Promise<boolean | void>;
 
 export class APIClient {
+  // TODO make rateLimitConfig configurable and only use the default values when none are provided
```

Comment:
nit: Austin recently asked that we start adding our GitHub username and a ticket number into the TODO comments..

Reply:
Yeah, I saw that and thought it was a good idea but hadn't thought to come back and update these. Good looking out!  Thanks!